content/advanced/330_servicemesh_using_appmesh/virtual_gateway_setup/testing_virtualgateway.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Testing Virtual Gateway"
+date: 2020-01-27T08:30:11-07:00
+weight: 80
+---
+
+Now it's time to test. Get the Loadbalancer endpoint that Virtual Gateway is exposed.
+    
+{{% notice info %}}
+It takes 3 to 5 minutes to set up the Load Balancer. You can go to Console and Go to Load Balancer and check if the state is `Active`.
+Do not proceed to next step until Load Balancer is `Active`.
+{{% /notice %}}
+
+```bash
+export LB_NAME=$(kubectl get svc ingress-gw -n prodcatalog-ns -o jsonpath="{.status.loadBalancer.ingress[*].hostname}") 
+ curl -v --silent ${LB_NAME} | grep x-envoy
+echo $LB_NAME
+```
+Check if the request to the Ingress Gateway is going from envoy by curl to the above Loadbalancer url
+{{< output >}}    
+> GET / HTTP/1.1
+> Host: db13be460b8648c4bXXXf.elb.us-west-2.amazonaws.com
+> User-Agent: curl/7.54.0
+> Accept: */*
+> 
+< HTTP/1.1 200 OK
+< content-type: text/html; charset=utf-8
+< content-length: 3783
+< x-amzn-trace-id: Root=1-5ff4c10e-71cca9a19486406b80eaa475
+< server: envoy
+< date: Tue, 05 Jan 2021 19:42:06 GMT
+< x-envoy-upstream-service-time: 3
+< server: envoy
+{ [1079 bytes data]
+* Connection #0 to host db13be460b8648c4bXXXf.elb.us-west-2.amazonaws.com left intact
+
+workshop:~/environment $ echo $LB_NAME
+
+db13be460b8648c4bXXXf.elb.us-west-2.amazonaws.com
+{{< /output >}} 
+
+Copy paste the above Loadbalancer endpoint in your browser and you should see the frontend application loaded as below.
+
+![fronteend](/images/app_mesh_fargate/ui1.png)
```

Comment:
nit, should this first screenshot be a full image, including the architecture? someone following along may be confused why their app looks different. I think once shown, it'd be fine to zoom in as you've done.

Reply:
I did not want to show the UI with architecture here as we have not yet completed the canary and the architecture shows canary as well. Did not want to confuse them.

Reply:
I updated with the arch diagram